portal/src/main/webapp/js/src/oncoprint/studyview.js

```diff
@@ -31,9 +117,46 @@ $(document).ready(function() {
 			})();
 			$('#outer_loader_img').hide();
 			$('#oncoprint #everything').show();
+                        
+                        var gene_data = response.toJSON();
+                        //get to process data from gene panel
+                        if(genepanelValues.genepaneldata.length>0){
+                            for(var i=0; i < gene_data.length; i++)
+                            {
```

Comment:
if you don't mind, please move brackets to the same line to keep with the style of the rest of the code